### DISCUSSION TITLE
AES OCB mode

### DISCUSSION THREAD

**Discussion by Kebechet** [bcgit/bc-csharp#386] [Q&A] [answered]
Hello, I would like to ask what version of OCB cipher mode do you use ?
OCB1, OCB2 or OCB3 ? Because I didnt find specified this information anywhere. And my concers are in case of OCB2 because that version had some vulnerabilities.

**Comment by peterdettman** [comment, marked as answer]
Our implementation is for the version specified in [RFC 7253](https://tools.ietf.org/html/rfc7253).

Quoting from the RFC:
`The scheme described here is called OCB3 in the 2011 paper describing the mode [..].`
